docs/dictionary/command/go.lcdoc

```diff
@@ -36,6 +36,14 @@ go back 7
 Example:
 go invisible stack "Preferences"
 
+Example:
```

Comment:
Could you also declare the variables `local tStackFile, tStackFolder` here please?